listenbrainz/webserver/__init__.py

```diff
@@ -43,7 +43,6 @@ def gen_app(config_path=None, debug=None):
     app = CustomFlask(
         import_name=__name__,
         use_flask_uuid=True,
-        use_debug_toolbar=True,
```

Comment:
This has been fixed in latest master too (changes like these become a bit distracting), please rebase once over master.